test/server/cards/01_SOR/units/AdmiralOzzelOverconfident.spec.ts

```diff
@@ -0,0 +1,77 @@
+describe('Admiral Ozzel, Overconfident', function() {
+    integration(function(contextRef) {
+        describe('Admiral Ozzel, Overconfident\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['atst', 'death-star-stormtrooper', 'maximum-firepower', 'liberated-slaves'],
+                        groundArena: ['admiral-ozzel#overconfident'],
+                        leader: 'grand-moff-tarkin#oversector-governor' // Making sure player1 has villainy aspect to be sure of cost later
+                    },
+                    player2: {
+                        groundArena: [{ card: 'wampa', exhausted: true }],
+                        spaceArena: ['ruthless-raider']
+                    }
+                });
+            });
+
+            it('should allow the controller to play an imperial unit from hand, which enters play ready, and allow each opponent to ready a unit', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.admiralOzzel);
+                expect(context.player1).toHaveEnabledPromptButtons(['Play an Imperial unit from your hand. It enters play ready', 'Attack']);
+
+                context.player1.clickPrompt('Play an Imperial unit from your hand. It enters play ready');
+                expect(context.admiralOzzel.exhausted).toBe(true);
+                expect(context.player1).toBeAbleToSelectExactly([context.atst, context.deathStarStormtrooper]);
```

Comment:
Can we add an expectation for the presence of the pass prompt

Reply:
I got it so that there's a 'Choose no target' button(had to edit SelectCardSystem a bit because it doesn't use the CardTargetResolver code). Does that work?

Reply:
Yep! There is also now a `toHaveChooseNoTargetButton()` expectation you can use